__tests__/hooks/pager.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { act, renderHook } from '@testing-library/react-hooks';
+import { User } from 'src/contrib/aidbox';
+import { usePager } from '../../hooks/pager';
+import { useService } from '../../hooks/service';
+import { success } from '../../libs/remoteData';
+import { getFHIRResources } from '../../services/fhir';
+
+jest.mock('../../services/fhir', () => ({ getFHIRResources: jest.fn() }));
+jest.mock('../../hooks/service', () => ({ useService: jest.fn() }));
+
+describe('Hook `usePager`', () => {
+    const resourceType = 'User';
+    const resourcesOnPage = 2;
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    describe('property `hasNext`', () => {
+        test("returns false when there's no next page", () => {
+            const data = success({
+                id: 'fakeID',
```

Comment:
Let's make tests in a more FHIR way. It will help us easily read the tests.
e.g. data = success({ resourceType: 'Bundle' }) without id (bundle is a virtual resource and used for list returning as a result